controllers/controller_common.go

```diff
@@ -118,3 +120,9 @@ func applyClusterRefLabel(cluster *v1beta1.KafkaCluster, labels map[string]strin
 func SetNewKafkaFromCluster(f func(k8sclient client.Client, cluster *v1beta1.KafkaCluster) (kafkaclient.KafkaClient, func(), error)) {
 	newKafkaFromCluster = f
 }
+
+func ObjectManagedByClusterRegistry(object v1.Object) bool {
```

Comment:
If this function is invoked only within the `controller` package  this can be made private.

Reply:
The `ObjectManagedByClusterRegistry` has been movbed to `pkg/util` as we need to use it for webhooks as well.